internal/projectinit/scaffolding/.vscode/settings.json

```
@@ -0,0 +1,3 @@
+{
+    "java.autobuild.enabled":false
+}
```

Uber nit! but we might want to format this file so it doesn't get reformatted in users projects?

```suggestion
{
  "java.autobuild.enabled": false
}
```